playwright/tests/page-objects/page-object.ts

```diff
@@ -3,14 +3,18 @@ import { Locator, Page, expect } from '@playwright/test';
 export class PageObject {
   constructor(readonly page: Page) {}
 
-  async executeCommand(command: string) {
+  async executeCommand(command: string, ...options: Array<string>) {
```

Comment:
is there a use case for more than one option?

Reply:
Yes, with an improvement I will make in the next commit, where I simply call `this.provideUserInput(option);` in the loop over `options`, something like:
https://github.com/axonivy/vscode-extensions/blob/38ecdbfc585e4396ae31e73f2a26df735b02d18b/playwright/tests/page-objects/explorer-view.ts#L76-L80

can be replaced with:
`await this.executeCommand('Axon Ivy: New Project', projectName, '', '', '');`

Reply:
very good 👍